src/hls-adapter.ts

```diff
@@ -126,6 +126,7 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
   private _nativeTextTracksMap: any = {};
   private _lastLoadedFragSN: number = -1;
   private _sameFragSNLoadedCount: number = 0;
+  private _firstHideTextTrack: boolean = true;
```

Comment:
Please use a more readable name.
What does this variable mean ?

Reply:
not sure how i can improve that, change to _isFirstSubtitleHide, the first time that captions is hided by the player. do you have any other suggestion?